From 180a65d0cfe510b2518ae61ad000b4962216320f Mon Sep 17 00:00:00 2001
From: lianglulu <2042213396@qq.cm>
Date: Tue, 31 May 2022 19:42:08 +0800
Subject: [PATCH 2/2] fix : the require config path mixed

- PWD represent buildroot path
- CURDIR represrnt output/build/linux-custom
---
 drivers/gpu/drm/gcnano-driver-6.4.3/Kbuild | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/gcnano-driver-6.4.3/Kbuild b/drivers/gpu/drm/gcnano-driver-6.4.3/Kbuild
index a2d64e66c..4218add6a 100644
--- a/drivers/gpu/drm/gcnano-driver-6.4.3/Kbuild
+++ b/drivers/gpu/drm/gcnano-driver-6.4.3/Kbuild
@@ -57,7 +57,7 @@
 # Linux build file for kernel HAL driver.
 #
 
-include $(PWD)/drivers/gpu/drm/gcnano-driver-6.4.3/config
+include $(CURDIR)/drivers/gpu/drm/gcnano-driver-6.4.3/config
 
 soc_vendor    := $(firstword $(subst -, ,$(SOC_PLATFORM)))
 soc_board     := $(lastword  $(subst -, ,$(SOC_PLATFORM)))
-- 
2.17.1

